spec/conformance/server.rb

```diff
@@ -95,20 +95,17 @@ def descriptor
   public_key_credential = WebAuthn::PublicKeyCredential.from_get(params, encoding: :base64url)
   expected_challenge = Base64.urlsafe_decode64(cookies["challenge"])
 
-  allowed_credentials = Credential.registered_for(cookies["username"]).map do |c|
-    { id: Base64.urlsafe_decode64(c.id), public_key: c.public_key, sign_count: c.sign_count }
+  user_credential = Credential.registered_for(cookies["username"]).detect do |uc|
+    uc.id == public_key_credential.id
   end
 
   public_key_credential.verify(
     expected_challenge,
-    allowed_credentials: allowed_credentials,
+    public_key: user_credential.public_key,
```

Comment:
I am seeing a regression for test AssertionResponse-Resp-3-F-4, the one testing the signature counter.

We need to pass `sign_count` here, don't we?

Reply:
Yes, thanks for catching that. I was still using version 1.1.4 of the tools where the bug documented in https://github.com/cedarcode/webauthn-ruby/issues/200 is present. Pushed a fix and verified it works correctly with 1.1.5 👍

Reply:
Oh right, the false positive from v1.1.4...

Thanks for the update!